### ISSUE TITLE
Moero Chronicle  (638160)

### ISSUE THREAD

**Issue by CygnusTerminal** [ValveSoftware/Proton#5506]
# Compatibility Report
- Name of the game with compatibility issues: Moero Chronicle
- Steam AppID of the game: 638160

## System Information
- GPU: GTX 1650 Laptop
- Driver/LLVM version: nvidia 495.46
- Kernel version: 5.16.1-zen1-1-zen
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/CygnusTerminal/4b3f848d9c279096371e51d24f97fd6d
- Proton version: 6.3-8

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-638160.log](https://github.com/ValveSoftware/Proton/files/7897404/steam-638160.log)

## Symptoms 
Game crashes at a black screen due to videos not loading. Tried various workarounds (proton-ge, m-youknowwhat, quartz, wmp10 and its codecs) possible to no avail, they only avoid the initial crashing but videos still wont play, making gameplay hard to follow since the game presents various plot points via video scenes.

## Reproduction
1. Launch the game
2. Observe the black screen
3. CTD

**Comment by ipr** [comment]
Referring to media-related issue: #1464

**Comment by Kaedras** [comment]
I tested the game using Proton experimental bleeding edge, Proton 7.0-4 and Proton-GE-7-37. The videos are working, but they are played in a seperate window, are stopped if the game window loses focus, and restarted if the game window regains focus. 

System Information:
- GPU: RTX 2070
- Drivers: nvidia 515.76
- Kernel: 6.0.0 (also tested with 5.19)
- Full report: https://gist.github.com/Kaedras/f3338529c214f2112496f270f9697eb4

[steam-638160.log](https://github.com/ValveSoftware/Proton/files/9725057/steam-638160.log)
![moero-chronicle-issue](https://user-images.githubusercontent.com/6783810/194320879-4b53d58e-2e64-49a6-aaa4-7ca1b8c42fd3.png)

**Comment by Kaedras** [comment]
I've done some more testing using Proton 7.0.4, experimental, and GE-Proton. Installing quartz using protontricks fixes the issue I mentioned before. There are some remaining issues however. The cutscene that plays when starting a new game just shows a static image and there is a weird transparency on the steam overlay. When switching to GE-Proton7-41 the video is displayed correctly, but there is no more sound on any video.

I've attached screenshots of the issues I mentioned and the new Proton log. Additionally, in case this information is of any use, I've updated my nvidia drivers to 525.53
![20221126183851_1](https://user-images.githubusercontent.com/6783810/204104896-8fec62ae-b4fa-47b9-98bc-9b790e6f5d52.jpg)

![20221126183933_1](https://user-images.githubusercontent.com/6783810/204104900-ada11a84-b2f2-48f9-b56d-d2bca69433aa.jpg)
[steam-638160.log](https://github.com/ValveSoftware/Proton/files/10096869/steam-638160.log)

**Comment by waspennator** [comment]
Game silently crashes out on my oled deck when launching on proton Experimental, downgraded to an older version of proton (8.0-5), stopped the crashing, but got stuck on a black screen (An Active movie window tab and the actual game tab) and could not click past it no matter what. I was able to get to the title screen and ingame with proton 5.0-10, but that broke with 5.13-6.

[steam-638160.log](https://github.com/ValveSoftware/Proton/files/14583446/steam-638160.log)

**Comment by waspennator** [comment]
Update, regression has been resolved and it seems to actually progress through the logos and intro fmvs to the title screen now (Sound only, screen is visually black when playing the fmvs till it hits the title.) Was even able to load up my old save from 2017

[steam-638160.log](https://github.com/ValveSoftware/Proton/files/14623981/steam-638160.log)

Update: Same occurs with the first fmv when starting a new game, managed to get past the black screen after a minute.

**Comment by waspennator** [comment]
Might as well repost here, game has regressed lately (around the new media source update), fmv no longer plays unless you manually tab onto the active movie tab each time an fmv is supposed to play. Game gets stuck on a black screen unless you do it.

[steam-638160.log](https://github.com/user-attachments/files/18105188/steam-638160.log)

**Comment by AJuujarvi** [comment]
@waspennator This was an intentional change to get it back to Proton 8.0-5 behavior. This change allows the video to be played with visible video on a Desktop compared to before. Before this change on Desktop, it was creating horrible audio static that a user might find damaging with some devices, or it would be crashing. While that may cause an additional step for Steam Deck users that understandably seems regressive, it prevents a bigger problematic behavior on Desktop. Ideally we will be able to resolve it so that the focus switching isn't needed + having the audio play correctly on Desktop.

**Comment by waspennator** [comment]
> [@waspennator](https://github.com/waspennator) This was an intentional change to get it back to Proton 8.0-5 behavior. This change allows the video to be played with visible video on a Desktop compared to before. Before this change on Desktop, it was creating horrible audio static that a user might find damaging with some devices, or it would be crashing. While that may cause an additional step for Steam Deck users that understandably seems regressive, it prevents a bigger problematic behavior on Desktop. Ideally we will be able to resolve it so that the focus switching isn't needed + having the audio play correctly on Desktop.

Retested it with latest steamos and proton Experimental bleeding edge and the fmv is playing perfectly fine now, the only issue I'm really noticing left now is the compile heart fmv sound cuts out halfway through

[steam-638160.log.txt](https://github.com/user-attachments/files/20719528/steam-638160.log.txt)

**Comment by waspennator** [comment]
Compile heart logo fmv is still cutting out halfway through audio wise
